src/client/layout/layout_object.hpp

```diff
@@ -479,6 +501,12 @@ namespace client_layout
     std::weak_ptr<LayoutObject> parent_;
     std::weak_ptr<LayoutObject> previous_;
     std::weak_ptr<LayoutObject> next_;
+    
+    /**
+     * The layer number for this layout object. 
+     * Layer 0 means no scrollable container ancestors, layer 1+ means inside scrollable containers.
```

Comment:
The description is incorrect, 0 means in the root scrollable container, because `LayoutView` is always a scrollable container.